Packs/FireEyeHX/Integrations/FireEyeHXv2/FireEyeHXv2.py

```diff
@@ -691,14 +691,35 @@ def get_token_request(self):
                 resp_type='response'
             )
         except Exception as e:
-            demisto.debug(f'Encountered an error for url {self._base_url}/token: {e}')
+            exception_str = str(e)
+            demisto.debug(f'Encountered an error for url {self._base_url}/token: {exception_str}')
```

Comment:
info level log message.
```suggestion
            demisto.info(f'Encountered an error for url {self._base_url}/token: {e}')
```